resources/shaders/dof_properties.hlsl

```diff
@@ -0,0 +1,2 @@
+static const float maxBokehSize = 18.f;
```

Comment:
Shouldn't this be user editable? (not sure)

Reply:
This should be a static variable as it perfectly fits the kernel method. Increasing it will result in a scuffed bokeh